internal/token.go

```diff
@@ -1236,7 +1236,13 @@ func (z *Tokenizer) readTagAttrExpression() {
 				if next == '/' {
 					panic("Block comments (//) are not allowed inside of expressions")
 				}
-				z.readCommentOrRegExp()
+				// Also stop when we hit a '}' character (end of attribute expression)
+				z.readCommentOrRegExp([]byte{'}'})
```

Comment:
Previously, we were only doing `z.readCommentOrRegExp()` which didn't know what to do with a single `/` character (it assumed it was part of a RegExp literal.

Now `readCommentOrRegExp` accepts additional characters that can be considered a boundary, like `}`.